pulumi/index.ts

```diff
@@ -24,16 +24,25 @@ export = async (): Promise<Outputs> => {
   } else {
     if (!config.rootDomainName) throw new Error('rootDomainName required')
 
+    const autoscaling = config.eks.autoscaling
+      ? {
+          enabled: config.eks.autoscaling.enabled,
+          minInstances: config.eks.autoscaling.minInstances,
+          maxInstances: config.eks.autoscaling.maxInstances,
+        }
+      : undefined
+
     const cluster = await EKSClusterLauncher.create(name, {
-      rootDomainName: config.rootDomainName,
-      instanceTypes: config.eks.instanceTypes,
-      numInstancesPerAZ: 2,
       allAZs: config.eks.allAZs,
-      region: config.eks.region,
+      autoscaling: autoscaling,
```

Comment:
We don't need to reassign the autoscaling object, can just use the object if defined or it will be undefined
```suggestion
      autoscaling: config.eks.autoscaling,
```

Reply:
Also remove object alongside this suggestion